src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -149,13 +150,15 @@ class JesBackend extends Backend with LazyLogging {
     val filesWithinExpressions = Try(
       call.task.outputs.map {
         taskOutput => taskOutput.expression.preevaluateExpressionForFilenames(scopedLookupFunction, engineFunctions)
-      }.flatMap({_.get})
+      }.flatMap({
```

Comment:
I revert it whenever it happens, but I don't get them all either.  I think we should try to modify the style to suit our preferences if possible.